From: Marco Nenciarini <mnencia@debian.org>
Date: Fri, 16 Sep 2011 00:55:02 +0200
Subject: fix mail_plugin_dir default

Description: Fix mail_plugin_dir default value in conf.d/10-mail.conf
Bug: #624294
---
 doc/example-config/conf.d/10-mail.conf |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/doc/example-config/conf.d/10-mail.conf b/doc/example-config/conf.d/10-mail.conf
index fe5cc31..4b918ed 100644
--- a/doc/example-config/conf.d/10-mail.conf
+++ b/doc/example-config/conf.d/10-mail.conf
@@ -200,7 +200,7 @@
 #auth_socket_path = /var/run/dovecot/auth-userdb
 
 # Directory where to look up mail plugins.
-#mail_plugin_dir = /usr/lib/dovecot
+#mail_plugin_dir = /usr/lib/dovecot/modules
 
 # Space separated list of plugins to load for all services. Plugins specific to
 # IMAP, LDA, etc. are added to this list in their own .conf files.
-- 
